server/sonar-server/src/main/java/org/sonar/server/platform/ServerSettingsImpl.java

```diff
@@ -35,17 +35,15 @@
  * <li>environment variables</li>
  * <li>system properties</li>
  * </ol>
- *
- * @since 2.12
  */
-public class WebServerSettings extends Settings implements ServerSettings {
+public class ServerSettingsImpl extends Settings implements ServerSettings {
 
   private final Properties properties;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) "properties" is the name of a field in "Settings". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2387)